### ISSUE TITLE
Localisation

### ISSUE THREAD

**Issue by luandro** [freeCodeCamp/devdocs#1168]
This tool is amazing, and essential for my workflow as a dev whose constantly offline.

I imagine it call also be amazing reference material for students. But English is not used by everyone, so it'd be great for non-english coders to use this as well.

Would this be possible?

**Comment by Bam92** [comment]
You're completely right. I am thinking also about this feature. This is possible because most of the docs are multilingual.

**Comment by simon04** [comment]
Here are some former tickets on this subject: https://github.com/freeCodeCamp/devdocs/issues/720, https://github.com/freeCodeCamp/devdocs/issues/794, https://github.com/freeCodeCamp/devdocs/issues/838. Unless someone comes up with a PR, this is unlikely to happen.

**Comment by stringa** [comment]
Can we please allow for this feature to be toggleable? Or allow someone to use the current system/setup?

**Comment by Bam92** [comment]
I think the system/setup can be good by default.
